packages/next/src/server/dev/hot-reloader-turbopack.ts

```diff
@@ -544,8 +544,9 @@ export async function createHotReloaderTurbopack(
     },
 
     // TODO: Figure out if socket type can match the NextJsHotReloaderInterface
-    onHMR(req, socket: Socket, head) {
+    onHMR(req, socket: Socket, head, callback) {
```

Comment:
callback arg (callback) in a callback (onHMR) might be confusing

```suggestion
    onHMR(req, socket: Socket, head, onUpgrade) {
```